.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+workflows:
+  version: 2
+  test-and-build:
+    jobs:
+      - test
+      - build:
+          requires:
+            - test
+jobs:
+  build:
+    working_directory: ~/prison-visits-integration-tests
+    docker:
+      - image: docker:17.09.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - type: shell
+        name: Push application Docker image
+        command: |
+          docker login -u $DOCKER_USER -p $DOCKER_PASS
+          docker build -t "stupidcodefactory/prison-visits-integration-tests:${CIRCLE_BRANCH}" .
+          docker push "stupidcodefactory/prison-visits-integration-tests:${CIRCLE_BRANCH}"
+
+
+  test:
+    parallelism: 1
+    working_directory: ~/prison-visits-integration-tests
+    docker:
+      - image: circleci/ruby:latest-browsers
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - restore_cache:
+          key: rails-demo-{{ checksum "Gemfile.lock" }}
+
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle
+
+      - type: shell
+        name: install firefox
+        command: |
+          wget -L "https://ftp.mozilla.org/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2" -O "firefox-$FIREFOX_VERSION.tar.bz2"
+          tar xjf "firefox-$FIREFOX_VERSION.tar.bz2"
+          sudo rm -rf /opt/firefox
+          sudo mv firefox /opt/
+          sudo ln -sf /opt/firefox/firefox /usr/bin/firefox
+
+
+      # Store bundle cache
+      - save_cache:
+          key: prison-visits-2-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+
+      - type: shell
+        command: |
+          wget https://github.com/mozilla/geckodriver/releases/download/v0.19.0/geckodriver-v0.19.0-linux64.tar.gz
+          tar -zxvf geckodriver-v0.19.0-linux64.tar.gz
+          sudo mv geckodriver /usr/local/bin/
+
+      - run: bin/rspec spec
+
+      # Save test results for timing analysis
+      - store_test_results:
+          path: coverage
```

Comment:
Do we need to save the screenshots here?

Reply:
oops that should be `screenshots`